setup/command/upgrade.go

```diff
@@ -0,0 +1,34 @@
+package command
+
+import (
+	"fmt"
+	"github.com/urfave/cli"
+)
+
+func (f *CommandFactory) Upgrade() cli.Command {
+	return cli.Command{
+		Name:      "upgrade",
+		Usage:     "upgrade a Layer0 instance to a new version",
+		ArgsUsage: "NAME VERSION",
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "force",
+				Usage: "skips confirmation prompt",
+			},
+		},
+		Action: func(c *cli.Context) error {
```

Comment:
You mention previously in the docs that major/minor version upgrades are not supported unless explicitly called out. I'm debating if we should have either logic or explicit help here to support that documentation call-out. (for example, help text that reiterates the doc's mention of explicit upgrade paths and/or logic that prevents certain upgrades from occurring)